ConnectorActionableMsgs/01 Build a basic Connector/Demos/01-card-webhook/Readme.md

```diff
@@ -1,44 +1,46 @@
+## Create a group and configure the webhook
+
+1. Open a browser and go to [Microsoft Outlook](https://outlook.office.com). Log in with your Office 365 credentials.
+1. Create a new group, or select an existing group.
+1. At the top-left of the screen, select the gear next to the member account. Choose **Connectors**.
 
-### Create Group and configure Webhook
+    ![Screenshot of group settings menu](../../Images/Exercise1-01.png)
 
-1. Open a browser and navigate to **https://outlook.office.com**. Log in with your Office 365 credentials.
-1. Create a new Group, or navigate to an existing Group.
-1. At the top-left of the screen, click the gear next to the member count. Choose **Connectors**.
+1. In the list of connectors, scroll down and select **Incoming Webhook**. Select **Add**.
 
-    ![](../../Images/Exercise1-01.png)
+    ![Screenshot of list of Connectors with incoming webhook highlighted](../../Images/Exercise1-02.png)
 
-1. In the list of Connectors, scroll down and then select **Incoming Webhook**. Click **Add**.
+1. Enter a name for the connector and select **Create**.
+1. The page will re-display, now including a URL for the connector. Use the icon next to the URL to copy it to the clipboard.
 
-    ![](../../Images/Exercise1-02.png)
+    ![Screenshot of incoming webhook with URL](../../Images/Exercise1-03.png)
 
-1. Enter a name for the connector and click **Create**.
-1. The page will re-display, now including a URL for the connector. Click the icon next to the URL to copy it to the clipboard.
+    > Note: The URL will be used several times in this exercise, as will the clipboard. We recommend pasting the URL into Notepad or other application.
 
-    ![](../../Images/Exercise1-03.png)
+1. Select **Done**. Then close the Connector and Group settings menus.
 
-    > Note: The URL will be used several times in this Exercise, as will the clipboard. We recommend pasting the URL into Notepad or other application.
+## Explore MessageCard playground
 
-1. Click **Done**. Then close the Connector and Group settings flyouts.
+1. In another browser tab or window, navigate to [MessageCard Playground](https://messagecardplayground.azurewebsites.net).
+1. The playground site provides for uploading a custom card definition or reviewing several samples. Use the **select a sample** dropdown to select a sample that closely matches your requirements.
 
-### Explore MessageCard Playground
+    ![Screenshot of message card sample](../../Images/Exercise1-04.png)
 
-1. In another browser tab or window, navigate to **https://messagecardplayground.azurewebsites.net**.
-1. The playground site provides for uploading a custom card definition or reviewing several samples. Use the **select a sample** dropdown to select a sample that closely matches your requirements. (The image in this lab uses the **Connectors reference example**, but that is not required.)
+    > Note: The image in this lab uses the **Connectors reference example**, but that is not required.
```

Comment:
That sample has been removed from the playground since Build 2018 - could we update to use another of the available samples?